sklearn/tree/export.py

```diff
@@ -778,3 +780,171 @@ def export_graphviz(decision_tree, out_file=None, max_depth=None,
     finally:
         if own_file:
             out_file.close()
+
+
+def export_ascii(decision_tree, feature_names=None,
```

Comment:
am I being too pedantic if I point out that feature/class names containing non-ascii characters will result in the output not being ascii? Should we call this `export_text` or `export_console`?

Reply:
I actually agree on this, I'll change it to `export_text`.